### ISSUE TITLE
Wrong parsing of RFC5424 if message contains a colon

### ISSUE THREAD

**Issue by nebulade** [solzimer/nsyslog-parser#1]
Through the docker syslog backend configured to do RFC5424 we see messages like:
```
'<30>1 2018-06-24T22:22:53Z my.test.com testapp 26599 testapp - This: contains two : colons'
```
Unfortunately the parsed message then contains only:
```
contains two : colons
```

Where the initial ` This:` is removed.

**Comment by solzimer** [comment]
Hi, @nebulade,

Thanks for the feedback. There is indeed a bug in this particular case, in which the parser interprets that token as a syslog BSD header element.

It has been fixed in [version 0.8.2](https://www.npmjs.com/package/nsyslog-parser)

**Comment by nebulade** [comment]
That is great news, thanks a lot for the fast reaction on this. I can confirm it is fixed for me now as well.
